src/backend/cdb/cdbfilerepresyncmanager.c

```diff
@@ -1132,15 +1132,17 @@ FileRepPrimary_RunResyncManager(void)
 				continue;
 			}
 		}
-		
+
+		FileRep_GetRelationPath(
```

Comment:
OK, the call to FileRep_GetRelationPath() was moved.
Referencing to PR description I want to ask: Why is it safe to use relFileNode here, but not safe to pass it to PersistentFileSysObj_ResynchronizeRefetch?

Reply:
It seems I misunderstood the code and relFileNode is OK in both versions.